.github/workflows/component-go.yaml

```diff
@@ -118,3 +123,30 @@ jobs:
         working-directory: "./examples/component/${{ matrix.example }}"
         run: |
           wash build
+
+      - name: run tests
+        working-directory: "./examples/component/${{ matrix.example }}"
+        run: go test .
```

Comment:
Could we change this to `go test ./...` here?

Reply:
Yeah we can. I just did this because the only test is at the top level